tensorflow_probability/python/distributions/truncated_normal.py

```diff
@@ -420,6 +420,19 @@ def _mode(self):
     bc_loc = tf.broadcast_to(loc, shp)
     return tf.clip_by_value(bc_loc, low, high)
 
+  def _quantile(self, p):
+    # TODO: This implementation is analytically correct, but might not
+    #  perform well in all cases. See
+    #  https://en.wikipedia.org/wiki/Truncated_normal_distribution#Generating_values_from_the_truncated_normal_distribution
+    #  for a discussion on alternatives.
+    loc, scale, low, high = self._loc_scale_low_high()
+    alpha = (low - loc) / scale
```

Comment:
We generally avoid greek for variable names. How about `std_low` and `std_high`?

Reply:
I'd love some clearer names for these, but I am unsure what would be appropriate. These are the same alpha and beta that are defined here: https://en.wikipedia.org/wiki/Truncated_normal_distribution

To me they just seem like convenient placeholders allowing for a more compact expression. What is your motivation for `std_low` and `std_high` in particular?

Reply:
`std_low` and `std_high` are used elsewhere in this file for this expression. It is also expressing that these are standard scores for `low` and `high`.

Reply:
I renamed the variables given the naming scheme you suggested.